Packs/Core/Playbooks/playbook-Identity_Analytics_-_Alert_Handling.yml

```diff
@@ -757,7 +757,7 @@ tasks:
       version: -1
       name: Get entity alerts by MITRE tactics
       description: |-
-        This playbook searches XDR alerts related to specific entities , on a given timeframe, based on MITRE tactics.
+        This playbook searches XDR alerts related to specific entities, on a given timeframe, based on MITRE tactics.
```

Comment:
Please add a loop for each input in case the alert.username and the actor_primary_username are not equal and are not 'NT AUTHORITY\SYSTEM